routes/router.js

```diff
@@ -87,12 +87,10 @@ module.exports = function(app, passport, path) {
 	  	function(req, res) {
 	  		console.log(JSON.stringify(req.user));
 	  		if (!req.user) {
-	  			// res.redirect('/login');
```

Comment:
So... here, I believe I have console.log('User not found!'); and I commented the res. out initially because I thought that it was unnecessary, because when you submit a login, you're already at the login page, so there is no need to redirect.  Hope that makes sense.  And I have no idea if this is appropriate to do.